pyproject.toml

```diff
@@ -41,7 +41,11 @@ pydantic = ">=1.9,<2.10"
 eval-type-backport = "0.2.0"
 curl_cffi = "0.6.2"
 pandoc = "*"
-pillow = "^11.0.0"
+rouge = "*"
+pillow = "^10.2.0"
+tree-sitter = "*"
+tree-sitter-python = "*"
+googletrans-py = "4.0.0"
```

Comment:
where did these dependencies used?

Reply:
Hi Wendong, the googletrans-py is used in the APIs defined for the API-Bank, while the tree-sitter and tree-sitter-python are used for evaluation of APIBench.